.github/workflows/ci.yml

```
@@ -0,0 +1,124 @@
+      with:
+        args: >
+          -Dsonar.organization=learnathon-By-Geeky-Solutions
+          -Dsonar.projectKey=Learnathon-By-Geeky-Solutions_skynet
+          -Dsonar.projectName=skynet
+          -Dsonar.javascript.lcov.reportPaths=backend/coverage/lcov.info,frontend/coverage/lcov.info
+          -Dsonar.coverage.exclusions=**/*.test.js,**/*.spec.js,**/__tests__/**,**/__mocks__/**
+          -Dsonar.javascript.node.maxspace=8192
+          -Dsonar.sonarcloud.automaticAnalysisDisabled=true
+          -Dsonar.analysis.automaticAnalysisConfigurationRun=false
+          -Dsonar.ci.autoconfig.disabled=true
+          -Dsonar.qualitygate.wait=true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)